client/components/dashboard/DashboardPage.js

```diff
@@ -75,8 +76,15 @@ class DashboardPage extends React.Component {
    * @return {type}            description
    */
   onSelect(pageNumber) {
-    const offset = (pageNumber - 1) * 15;
-    this.props.getUsers(offset);
+    const searchStatus = this.props.currentState.searchParams;
+    if (searchStatus.isSearch) {
+      const offset = (pageNumber - 1) * 15;
```

Comment:
This code is unclear. What does this calculation do. Either use comments to explain or move it to a function with a clear name